test/slack/handlers_test.exs

```diff
@@ -123,6 +123,35 @@ defmodule Slack.HandlersTest do
     assert new_slack.channels["123"].members == []
   end
 
+  test "group_joined event should add group" do
+    {:ok, new_slack} = Handlers.handle_slack(
+      %{type: "group_joined", channel:
+        %{id: "G123", members: ["U123", "U456"]}},
```

Comment:
Can we put the ending backets and curlies on their own lines? to help readability?